pybytes/README.md

```diff
@@ -4,7 +4,21 @@
 
 Please note that these examples only work when using the `pybytes` firmware.
 
-# Pytrack
-- To run pytrack on individuals module user will need to create one pybytes_config.json file on root which usually contains data related to device e.g Network preferance, device id, server, username, password etc.
+# Pytrack, Pysense, Pyscan
+- To run pytrack on individuals module you will need to create one pybytes_config.json file on root which usually contains data related to device e.g Network preferance, device id, server, username, password etc.
 
-- User needs to upload all the files to module in order to get data on pybytes dashboard.
+- You will needs to upload all the files to module in order to get data on pybytes dashboard.
+
+Note: Note: For using Pyscan, you need to upload either MFRC630.mpy or MFRC630.py.
+It is always recommended to use MFRC630.mpy as it takes less space on your module.
+MFRC630.py is supported for newer version only e.g. WiPy3, LoPy4, GPy, FiPy
```

Comment:
Could we clarify that it's supported on the other boards as well, but due to memory restrictions, you can't use the `.py` version? But I think this could be removed